packages/core/src/components/ozon-content/ozon-content-mapper.tsx

```diff
@@ -103,8 +103,17 @@ export class Mapper {
   }
 
   transform(xml: string, context: OzonContentContext): JSX.Element {
+    if (!xml) {
+      return <Fragment />;
+    }
+
     if (!this.cache || this.cache.xml !== xml) {
```

Comment:
Issue maken voor optimaliseren van cache in OzonContentMapper.

Reply:
https://github.com/orgs/dso-toolkit/projects/4/views/4?pane=issue&itemId=33392219